File: EmojiPanel/src/main/java/com/binlee/emoji/model/WrappedEmoji.java
package com.binlee.emoji.model;

import java.io.Serializable;
import java.util.List;

/**
 * Created on 19-7-30.
 *
 * @author leebin
 */
public class WrappedEmoji implements Serializable {
    
    private static final long serialVersionUID = 3392703176362269326L;
    
    public Emoji mEmoji;
    public EmojiGroup mGroup;
    public List<Emoji> mEmojiList;
}

File: EmojiPanel/src/main/java/com/binlee/emoji/model/WrappedEmoji.kt
package com.binlee.emoji.model

import java.io.Serializable

/**
 * Created on 19-7-30.
 *
 * @author leebin
 */
class WrappedEmoji : Serializable {

    var mEmoji: Emoji? = null
    var mGroup: EmojiGroup? = null
    var mEmojiList: List<Emoji>? = null

    companion object {
        private const val serialVersionUID = 3392703176362269326L
    }
}